docs/sphinx/examples/python/tutorials/Shors.ipynb

```diff
@@ -414,35 +424,35 @@
    "id": "0c644786",
    "metadata": {},
    "source": [
-    "Verify in the code block below that the kernels defined above do in fact carry out multiplication and exponentiation with $a=5$ and $N=21$. "
+    "Verify in the code block below that the kernels defined above do in fact carry out multiplication and exponentiation with $a=5$ and $N=21$ by changing the `iterations` variable below. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 54,
    "id": "e93fcf39",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "     ╭───╮╭───╮   ╭───╮      ╭───╮      ╭───╮      ╭───╮      \n",
-      "q0 : ┤ x ├┤ x ├─╳─┤ x ├────╳─┤ x ├────╳─┤ x ├────╳─┤ x ├────╳─\n",
-      "     ╰───╯╰───╯ │ ╰───╯    │ ╰───╯    │ ╰───╯    │ ╰───╯    │ \n",
-      "q1 : ───────────┼──────────┼──────────┼──────────┼──────────┼─\n",
-      "     ╭───╮      │ ╭───╮    │ ╭───╮    │ ╭───╮    │ ╭───╮    │ \n",
-      "q2 : ┤ x ├──╳───┼─┤ x ├─╳──┼─┤ x ├─╳──┼─┤ x ├─╳──┼─┤ x ├─╳──┼─\n",
-      "     ╰───╯  │   │ ╰───╯ │  │ ╰───╯ │  │ ╰───╯ │  │ ╰───╯ │  │ \n",
-      "q3 : ───────┼───┼───────┼──┼───────┼──┼───────┼──┼───────┼──┼─\n",
-      "     ╭───╮  │   │ ╭───╮ │  │ ╭───╮ │  │ ╭───╮ │  │ ╭───╮ │  │ \n",
-      "q4 : ┤ x ├──╳───╳─┤ x ├─╳──╳─┤ x ├─╳──╳─┤ x ├─╳──╳─┤ x ├─╳──╳─\n",
-      "     ╰───╯        ╰───╯      ╰───╯      ╰───╯      ╰───╯      \n",
+      "     ╭───╮╭───╮   ╭───╮      ╭───╮      ╭───╮      \n",
+      "q0 : ┤ x ├┤ x ├─╳─┤ x ├────╳─┤ x ├────╳─┤ x ├────╳─\n",
+      "     ╰───╯╰───╯ │ ╰───╯    │ ╰───╯    │ ╰───╯    │ \n",
+      "q1 : ───────────┼──────────┼──────────┼──────────┼─\n",
+      "     ╭───╮      │ ╭───╮    │ ╭───╮    │ ╭───╮    │ \n",
+      "q2 : ┤ x ├──╳───┼─┤ x ├─╳──┼─┤ x ├─╳──┼─┤ x ├─╳──┼─\n",
+      "     ╰───╯  │   │ ╰───╯ │  │ ╰───╯ │  │ ╰───╯ │  │ \n",
+      "q3 : ───────┼───┼───────┼──┼───────┼──┼───────┼──┼─\n",
+      "     ╭───╮  │   │ ╭───╮ │  │ ╭───╮ │  │ ╭───╮ │  │ \n",
+      "q4 : ┤ x ├──╳───╳─┤ x ├─╳──╳─┤ x ├─╳──╳─┤ x ├─╳──╳─\n",
+      "     ╰───╯        ╰───╯      ╰───╯      ╰───╯      \n",
       "\n",
-      "Measurement results: { 10001:200 }\n",
+      "Measurement results in big-endian notation: { 00001:200 }\n",
       "\n",
-      "For x =5, 5^x mod 21 = 17\n",
-      "For x =5,the computed result of the circuit is 17\n"
+      "For x = 4, 5^x mod 21 = 16\n",
+      "For x = 4, the computed result of the circuit is 16\n"
```

Comment:
Big endian is defined as having the most significant bit on the left-most character in the string, so `00001` would be `1`, not 16, no?

Reply:
ugh. endian-notation is haunting me this week.  Yes, I needed to change the circuit for modular multiplication (which was based on the wrong endian).  I'll submit the PR for this fix.  thanks for catching this.